### PR TITLE
sumo 0.24.0

### PR DESCRIPTION
**Pull request by alexvoronov**

### CODE REVIEW

File: sumo.rb [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
   homepage "https://sourceforge.net/projects/sumo/"
-  url "https://downloads.sourceforge.net/project/sumo/sumo/version%200.23.0/sumo-all-0.23.0.tar.gz"
-  sha256 "8a6354a11717cdff2f3f247239fb55472ea57268b392e5e442777a1c05d92299"
-
-  bottle do
-    cellar :any
-    sha256 "fc42a513a71d13b54d230ac0bc26cbc96a0aa2323ee434c4313ef28e4e1f3217" => :yosemite
-    sha256 "13ab2b7e6d904ee4009162a7a8a3c6038d3aec2a48c4407a5093b5aa729ab9e4" => :mavericks
-    sha256 "4b2e04b94e1a95cc5759d422402b34c9534cbc8873911038fe41a2ee0fbc271d" => :mountain_lion
-  end
```
Comment (dpo) [inline sumo.rb]: Please don't remove the `bottle` section. It will be updated automatically.

**Comment by dpo** [comment]
See #2742.

**Comment by alexvoronov** [comment]
Thanks for pointing me to #2742. I see it was never finished. I'm not sure if I can reuse that pull request and base my pull request on top of it, so I just will base my pull request on master. I fixed fox to use x11 dependency as commented in #2742. 

By the way, the `default` check result "No test results found". There are tests in the formula, both in the install phase and in `test do`. What can I add to pass that check?

**Comment by dpo** [comment]
The current error is
```
Error: No available formula for fox 
```
which should be fix if the dependency points to `homebrew/x11/fox` as I indicated.

**Comment by dpo** [comment]
```
==> brew audit homebrew/science/sumo
==> FAILED
Error: 1 problem in 1 formula
homebrew/science/sumo:
 * Stable: Use https://downloads.sourceforge.net to get geolocation (url is http://sourceforge.net/projects/sumo/files/sumo/version%200.24.0/sumo-all-0.24.0.tar.gz).
```

**Comment by alexvoronov** [comment]
Sorry, changed URL and forgot to re-do `brew audit`. Hope everything is fine now :)

**Pull request closed without merging**